Application Settings/DependentListController.cls

```diff
@@ -0,0 +1,25 @@
+public with sharing class DependentListController {
+    
+    public String selected{get;set;}
+    public List<SelectOption> countryOptions{get;set;}
+    public String option{get;set;}
```

Comment:
what is this option? which option. Please change this variable name to some meaningful name